tests/forms_tests/field_tests/test_disabledfield.py

```diff
@@ -0,0 +1,33 @@
+from django.forms import Field, Form
+from django.test import SimpleTestCase
+
+
+class TestForm(Form):
+    disabled_empty_initial = Field(disabled=True, required=False)
+    disabled_initial = Field(disabled=True, initial='Test')
+
+
+class DisabledFieldTest(SimpleTestCase):
+    def test_disabled_field(self):
+        disabled_field = Field(disabled=True)
+        initial = disabled_field.initial
+        self.assertFalse(disabled_field.has_changed(initial, ''))
+
+        disabled_field = Field(disabled=True, initial='Test')
+        initial = disabled_field.initial
+        self.assertFalse(disabled_field.has_changed(initial, ''))
+
+    def test_disabled_field_in_unbound_form(self):
```

Comment:
I'm uncertain about the bottom two tests, they're already passing without the fix. If they're simply to add additional test coverage, we usually add such tests in a separate commit to make it more clear what the fix is actually fixing.

In case you didn't seem them, there are some other tests for `disabled=True` in `tests/test_forms.py`.